source/extensions/filters/network/kafka/protocol_code_generator/request_codec_request_test_cc.j2

```diff
@@ -12,6 +12,7 @@
 #include "extensions/filters/network/kafka/external/requests.h"
 #include "extensions/filters/network/kafka/request_codec.h"
 
+#include "test/extensions/filters/network/kafka/buffer_based_test.h"
```

Comment:
Can you move this to the tests folder? Same below? I didn't catch this before but all test code should be in the test folder.